Potestas/Potestas/Observations/Comparers/EqualityComparers/BaseEqualityComparer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+
+namespace Potestas.Observations.Comparers.EqualityComparers
+{
+    public abstract class BaseEqualityComparer<T> : EqualityComparer<T> where T: IEnergyObservation
+    {
+        public bool? BaseEqualityCompare(T xObservation, T yObservation)
```

Comment:
if the name will be Equals here, then the method `Equals` from the base abstract class `EqualityComparer` will be hidden and I can not override it here since my return type is `bool?`. 
I override this method in classes: `EstimatedValueEqualityComparer, ObservationPointEqualityComparer, ObservationTimeEqualityComparer`  and renaim methods to DefaulValueEquals and DefaultValueOrderCompare 

https://github.com/ChristinaGolovach/.NET.Mentoring.L1/pull/1/commits/1dcade325f0ba826f513eb47c7b0e93d0e6a204b